tests/store/model_registry/test_file_store.py

```diff
@@ -1307,13 +1423,11 @@ def predict(self, context, model_input):
     with mlflow.start_run():
         mlflow.log_param("A", "B")
 
-    models = store.search_registered_models(max_results=10)
-    assert len(models) == 2
-    assert models[0].name == "model1"
-    assert models[1].name == "model2"
-    mv1 = store.search_model_versions("name = 'model1'")
-    assert len(mv1) == 2
-    assert mv1[0].name == "model1"
-    mv2 = store.search_model_versions("name = 'model2'")
-    assert len(mv2) == 1
-    assert mv2[0].name == "model2"
+        models = store.search_registered_models(max_results=10)
+        assert len(models) == 2
+        assert models[0].name == "model1"
+        assert models[1].name == "model2"
+        mv1 = store.search_model_versions("name = 'model1'", max_results=10)
+        assert len(mv1) == 2 and mv1[0].name == "model1"
```

Comment:
```suggestion
        assert len(mv1) == 2
        assert mv1[0].name == "model1"
```

Assertion on multiple conditions makes debugging difficlt. It should be broken down.